src/shorah/amplicon.py

```diff
@@ -325,8 +326,8 @@ def main(args):
     ref_seq = list(SeqIO.parse(in_fasta, 'fasta'))[0]
     ref_name = ref_seq.id
```

Comment:
Would it be easier and also add some robustness to just clean the reference id from special characters?  Something like `ref_name = re.sub(r'[^\w\s]', '-', ref_seq.id)`?